scripts/install.sh

```diff
@@ -1,7 +1,18 @@
 #!/bin/bash
+
+debug="$DEBUG"
+if [[ "$1" == "--debug" ]]; then
+    debug="true"
+fi
```

Comment:
Why have this flag in addition to the environment variable?

Reply:
The environment variable is necessary the way the docker build action works but flags are more natural on the cli IMO.

I can drop it though if you'd rather not have it.